fabric/src/test/java/TestRegexs.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+import com.wynntils.features.chat.GuildRankReplacementFeature;
+import com.wynntils.features.ui.BulkBuyFeature;
+import com.wynntils.models.items.annotators.gui.ArchetypeAbilitiesAnnotator;
+import java.lang.reflect.Field;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestRegexs {
```

Comment:
Why put it in fabric and not common?

Reply:
@magicus we junit test in fabric because of loaders. Just like styledtext/customcolor

Reply:
Hm. I assume we could have the tests in common, even if we are just able to run the tests from fabric currently? I think it feels odd to put generic code in the fabric folder. If we already do so, I'd argue that it is wrong.

Reply:
I am not sure. When we added it it was accepted as is, so we never tried it.